install.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+#
+# Script definetely inspired by
+#
+# https://github.com/nvm-sh/nvm/blob/master/install.sh
+# https://github.com/heroku/cli/blob/master/install-standalone.sh
+#
+
+{
+    echoerr() { echo "$@" 1>&2; }
+
+    try_profile() {
+        if [ -z "${1-}" ] || [ ! -f "${1}" ]; then
+            return 1
+        else
+            echo $1
+        fi
+    }
+
+    find_profile() {
+        local DETECTED_PROFILE
+        if [ "${SHELL#*bash}" != "$SHELL" ]; then
```

Comment:
This is doing the following: it removes everything up to and including the word "bash" from $SHELL (which in my case is /bin/bash). It actually deletes the shortest match of `*bash`.
This means that it will always be different from $SHELL in a bash system. 

Sure does the trick but it is kinda confusing. The same stands also for the zsh profile.
Why not checking if "bash" substring exists in then $SHELL, like in the case of "arm"*